core/src/lib.rs

```diff
@@ -291,6 +292,45 @@ impl Schema {
     pub fn identifies_attribute(&self, x: &NamespacedKeyword) -> bool {
         self.get_entid(x).map(|e| self.is_attribute(e)).unwrap_or(false)
     }
+
+    pub fn as_edn_value(&self) -> edn::Value {
+        let mut all_values = Vec::new();
+        for (entid, attribute) in &self.schema_map {
+            let mut attribute_map: BTreeMap<edn::Value, edn::Value> = BTreeMap::default();
+            attribute_map.insert(edn::Value::NamespacedKeyword(NamespacedKeyword::new("db", "id")), edn::Value::Keyword(Keyword::new(entid.to_string())));
```

Comment:
These values are all (lazily) declared in `src/db/values.rs`.  Let's move that to `core` and use them here.

You should find that you only need those values; anything else reveals a mismatch between what you're doing and what the system is doing elsewhere, and we should figure out what it is.